modules/apps/object/object-service/src/main/resources/resource-actions/default.xml

```diff
@@ -30,9 +30,12 @@
 		<weight>1</weight>
 		<permissions>
 			<supports>
+				<action-key>ADD_DISCUSSION</action-key>
 				<action-key>DELETE</action-key>
+				<action-key>DELETE_DISCUSSION</action-key>
 				<action-key>PERMISSIONS</action-key>
 				<action-key>UPDATE</action-key>
+				<action-key>UPDATE_DISCUSSION</action-key>
```

Comment:
Hi @seltong,

Set DELETE_DISCUSSION and UPDATE_DISCUSSION in guest-unsupported

thanks!